pkg/crc/config/settings.go

```diff
@@ -92,13 +93,15 @@ func RegisterSettings(cfg *Config) {
 	cfg.AddSetting(EmergencyLogin, false, ValidateBool, SuccessfullyApplied,
 		"Enable emergency login for 'core' user. Password is randomly generated. (true/false, default: false)")
 
-	// Shared directories configs
 	if runtime.GOOS == "windows" {
+		// Shared directories configs
 		cfg.AddSetting(SharedDirPassword, Secret(""), validateString, SuccessfullyApplied,
 			"Password used while using CIFS/SMB file sharing (It is the password for the current logged in user)")
-
 		cfg.AddSetting(EnableSharedDirs, false, validateSmbSharedDirs, SuccessfullyApplied,
 			"Mounts host's user profile folder at '/' in the CRC VM (true/false, default: false)")
+		// Setting to enable access to CRC vm from wsl
+		cfg.AddSetting(WSLHostAccess, false, ValidateBool, SuccessfullyApplied,
```

Comment:
is it network access or host access?

Reply:
should be `WSLNetworkAccess` updated now